app/models/user.rb

```diff
@@ -6,6 +6,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
+  belongs_to :draw
```

Comment:
yeah, we should have a discussion about this before break, I think. I'll open up a new issue.